exec/java-exec/src/main/java/org/apache/drill/exec/server/Drillbit.java

```diff
@@ -207,6 +210,21 @@ private void javaPropertiesToSystemOptions() {
   }
 
   /**
+   * If cluster version is the same as default, update option to current drillbit version.
+   */
+  private void checkAndUpdateClusterVersionOption(OptionManager optionManager, DrillbitEndpoint drillbitEndpoint) {
+    OptionValue versionOption = optionManager.getOption(ExecConstants.CLUSTER_VERSION);
+    OptionValidator validator = SystemOptionManager.getValidator(ExecConstants.CLUSTER_VERSION);
+    if (versionOption.equals(validator.getDefault())) {
+      optionManager.setOption(OptionValue.createOption(
```

Comment:
@sudheeshkatkam 
this was done on purpose [1]. Version is set as system option so administrator can update it when cluster upgrade was done. This will help administrator to detect drillbits that were not updated.

[1] https://issues.apache.org/jira/browse/DRILL-4604?focusedCommentId=15241083&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15241083